### DISCUSSION TITLE
Welcome to InvoBook Discussions!

### DISCUSSION THREAD

**Discussion by Hasnayeen** [iluminar/goodwork#1244] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mrandyyp** [comment]
hy, thanks for share the code, may I know the email and user password?

**Reply by Hasnayeen** [reply]
you've to create an account before you can login
